rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRestInterface.java

```diff
@@ -488,6 +495,47 @@ Object getNodeMBeanHistory(@HeaderParam("sessionid") String sessionId, @QueryPar
             throws InstanceNotFoundException, IntrospectionException, ReflectionException, IOException,
             NotConnectedException, MalformedObjectNameException, NullPointerException, MBeanException;
 
+    /**
+     * Return the statistic history of a list of nodes contained in the node RRD database,
+     * without redundancy, in a friendly JSON format.
+     *
+     * @param sessionId a valid session
+     * @param nodesJmxUrl a list of mbean server urls
+     * @param objectName name of mbean
+     * @param attrs set of mbean attributes
+     * @param range a String of 5 chars, one for each stat history source, indicating the time range to fetch
+     *      for each source. Each char can be:<ul>
+     *            <li>'a' 1 minute
+     *            <li>'m' 10 minutes
+     *            <li>'n' 5 minutes
+     *            <li>'t' 30 minutes
+     *            <li>'h' 1 hour
+     *            <li>'j' 2 hours
+     *            <li>'k' 4 hours
+     *            <li>'H' 8 hours
+     *            <li>'d' 1 day
+     *            <li>'w' 1 week
+     *            <li>'M' 1 month
+     *            <li>'y' 1 year</ul>
+     * @return a JSON object containing a key for each source
+     * @throws InstanceNotFoundException
+     * @throws IntrospectionException
+     * @throws ReflectionException
+     * @throws IOException
+     * @throws MalformedObjectNameException
+     * @throws NullPointerException
+     * @throws NotConnectedException
```

Comment:
There are rules which I follow when I write Java Docs. 
I never leave @throws, @param, or @return without comments - cause it is only make code longer.